dds/idl/langmap_generator.cpp

```diff
@@ -1370,6 +1370,14 @@ struct Cxx11Generator : GeneratorBase
     helpers_[HLP_FIXED_CONSTANT] = "IDL::Fixed_T";
   }
 
+  static void gen_cxx11_typecodes(const std::string& type) {
```

Comment:
```suggestion
  static void gen_typecode_ptrs(const std::string& type)
  {
```
This is in a class called `Cxx11Generator`